webapp/app.py

```diff
@@ -1,29 +1,40 @@
-import flask
-
 from canonicalwebteam.flask_base.app import FlaskBase
+
+from webapp.database import db
 from webapp.decorators import token_required
+from webapp.tokens.cli import register_token_cli
+from webapp.tokens.flask import register_token_views
 
 
 app = FlaskBase(__name__, "assets.ubuntu.com")
 
 
+register_token_cli(app)
+register_token_views(app)
+
+
 @app.route("/")
 @token_required
 def index():
     return {}, 200
 
 
+@app.errorhandler(400)
 @app.errorhandler(401)
-def error_401(exception=None):
-    return {"status": "401", "message": str(exception)}, 401
-
-
+@app.errorhandler(403)
 @app.errorhandler(404)
-def error_404(exception=None):
-    return {"status": "404", "message": "Resource not found"}, 404
+def error_handler(exception=None):
+    code = getattr(exception, "code")
+    return {"status": f"{code}", "message": str(exception)}, code
 
 
 @app.errorhandler(500)
 def error_500(exception=None):
-    flask.current_app.extensions["sentry"].captureException()
+    app.extensions["sentry"].captureException()
```

Comment:
what if it not configured? For example locally.
I think you need to add `if not app.testing:`

Reply:
Why couldn't you do something like `if 'sentry' in app.extensions`?

As I've mentioned a number of times before, I really consider explicitly changing your app based on whether you're "testing" or not to be a pretty bad antipattern.

Reply:
I think it can just be as is since captureException() will simply not push if `SENTRY_DSN` is not configured(to my understanding) and flask-base always installs raven.